src/components/VirtualList/VirtualList.tsx

```diff
@@ -14,6 +14,12 @@ import * as FilterHelper from '../FilterList/FilterHelper';
 import * as Sorts from '../../pages/Overview/Sorts';
 import { StatefulFilters } from '../Filters/StatefulFilters';
 
+// ******************************
+// VirtualList and its associated classes are deprecated and not to be used for new
```

Comment:
I'd change this comment for something more technical with more info.

VirtualList and its associated classes are intented to be used for main list pages: Applications, Workloads, Services and Istio Config. They share common style and filter integration, so they have some limitations when you want to use it for other scenarios that may require different personalization (columns style, or layout).

For secondary list rendered inside a detail page consider to use the Table component as used in WorkloadServices, WorkloadPods, ServiceInfoWorkload, IstioConfigSubList, ExperimentListContainer or TrafficListComponent.